src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/DomainEventSubscriber.kt

```diff
@@ -29,13 +29,12 @@ class DomainEventSubscriber(
     val event = gson.fromJson(requestJson, Event::class.java)
     val prisonerReceivedEvent = gson.fromJson(event.Message, PrisonerReceivedEvent::class.java)
 
-    log.info("Domain event received: {}", prisonerReceivedEvent.eventType)
-
     when (prisonerReceivedEvent.eventType) {
       "prison-offender-events.prisoner.received" ->
         restrictedPatientCleanup.deleteRestrictedPatientOnExternalMovementIntoPrison(
           prisonerReceivedEvent.additionalInformation.nomsNumber
         )
+      else -> log.warn("Unexpected domain event received: {}", prisonerReceivedEvent.eventType)
```

Comment:
I'll remove the filter to the `restricted-patients.patient.removed` in cloud platform so we should't get any of these warnings.